src/org/usfirst/frc4904/robot/autonomous/strategies/AutonVomit.java

```diff
@@ -0,0 +1,37 @@
+package org.usfirst.frc4904.robot.autonomous.strategies;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.BallOuttake;
+import org.usfirst.frc4904.standard.commands.Cancel;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisMoveDistance;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisTurn;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.WaitCommand;
+
+public class AutonVomit extends CommandGroup {// Vomit autonomously
+	public static final double DISTANCE_START_TO_POINT = 67.5 * RobotMap.Constant.RobotMetric.WHEEL_ENCODER_PPR / RobotMap.Constant.RobotMetric.WHEEL_CIRCUMFERENCE;
+	public static final double DISTANE_POINT_TO_BOILER = 73.18 * RobotMap.Constant.RobotMetric.WHEEL_ENCODER_PPR / RobotMap.Constant.RobotMetric.WHEEL_CIRCUMFERENCE;
+	public static final double DISTANCE_POINT_TO_BASELINE = -15 * RobotMap.Constant.RobotMetric.WHEEL_ENCODER_PPR / RobotMap.Constant.RobotMetric.WHEEL_CIRCUMFERENCE;
+	public static final double BOILER_TURN_ANGLE = 135;
+	
+	public static enum Alliance {
```

Comment:
You may want to keep this in order to maintain the 1 and -1 values, but this enum already exists somewhat in `DriverStation.Alliance` (wpilib), which is the enum that `DriverStation.getInstance().getAlliance()` returns.